src/components/toolbar/toolbar.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Component,
+  ChangeDetectionStrategy
+} from "angular2/core";
+import {Input} from "angular2/core";
+import {Renderer} from "angular2/core";
+import {ElementRef} from "angular2/core";
+
+@Component({
+  selector: 'md-toolbar',
+  inputs: ['color'],
+  templateUrl: './components/toolbar/toolbar.html',
+  styleUrls: ['./components/toolbar/toolbar.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MdToolbar {
+
+  private color_: string;
+
+  constructor(private elementRef: ElementRef, private renderer: Renderer) {}
+
+  set color(value: string) {
+    if (this.color_) {
+      this.renderer.setElementClass(this.elementRef.nativeElement, 'md-' + this.color_, false);
```

Comment:
This changed in beta.1 https://github.com/angular/angular/blob/master/CHANGELOG.md#200-beta1-catamorphic-involution-2016-01-08 (last line)

> renderer.setElementProperty, ..setElementStyle, ..setElementAttribute now take a native element instead of an ElementRef
Renderer interface now operates on plain native nodes, instead of RenderElementRefs or RenderViewRefs